bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/camera/CameraAddress.java

```diff
@@ -52,8 +53,10 @@ public boolean isVpnURLChanged(String vpnURL) {
 
     @Override
     public boolean equals(@Nullable Object object) {
-        if (this == object) return true;
-        if (object == null || getClass() != object.getClass()) return false;
+        if (this == object)
+            return true;
+        if (object == null || getClass() != object.getClass())
+            return false;
```

Comment:
Shouldn't if statements have brackets? Are brackets not enforced by spotless?

Reply:
Probably not. But changing that is out-of-scope of this PR.